README.md

```diff
@@ -62,6 +62,8 @@ Need:
 
 - Laravel 3: <http://maxoffsky.com/code-blog/history-of-laravel-php-framework-eloquence-emerging/>
 
+- Phalcon: <http://blog.phalconphp.com/search/release>
```

Comment:
You should say Phalcon 1: <http://blog.phalconphp.com/post/45916328008/phalcon-1-0-released> , so people can see the exact date of release. 1.x series are all same version. 2.x goes to next year. Please see other frameworks like Laravel, Cake, Solar / Aura how they are mentioned.